doc/hardware/porting/board_porting.rst

```diff
@@ -34,39 +93,28 @@ You can visualize the hierarchy like this:
 Here are some examples. Notice how the SoC series and family levels are
 not always used.
 
-.. list-table::
-   :header-rows: 1
-
-   * - Board
-     - SoC
-     - SoC series
-     - SoC family
-     - CPU core
-     - Architecture
-   * - :ref:`nrf52dk/nrf52832 <nrf52dk_nrf52832>`
-     - nRF52832
-     - nRF52
-     - Nordic nRF5
-     - Arm Cortex-M4
-     - Arm
-   * - :ref:`frdm_k64f <frdm_k64f>`
-     - MK64F12
-     - Kinetis K6x
-     - NXP Kinetis
-     - Arm Cortex-M4
-     - Arm
-   * - :ref:`stm32h747i_disco <stm32h747i_disco_board>`
-     - STM32H747XI
-     - STM32H7
-     - STMicro STM32
-     - Arm Cortex-M7
-     - Arm
-   * - :ref:`rv32m1_vega_ri5cy <rv32m1_vega>`
-     - RV32M1
-     - (Not used)
-     - (Not used)
-     - RI5CY
-     - RISC-V
+.. table::
+
+   +--------------------------------------------------+-----------------+-------------+---------------+---------------+----------------+--------------+
+   | Board                                            | Identifier      | SoC         | SoC Series    | SoC family    | CPU core       | Architecture |
+   +==================================================+=================+=============+===============+===============+================+==============+
+   | :ref:`nrf52dk <nrf52dk>`                         | /nRF52832       | nRF52832    | nRF52         | Nordic nRF5   | Arm Cortex-M4  | Arm          |
+   +--------------------------------------------------+-----------------+-------------+---------------+---------------+----------------+--------------+
+   | :ref:`frdm_k64f <frdm_k64f>`                     |                 | MK64F12     | Kinetis K6x   | NXP Kinetis   | Arm Cortex-M4  | Arm          |
+   +--------------------------------------------------+-----------------+-------------+---------------+---------------+----------------+--------------+
+   | :ref:`stm32h747i_disco <stm32h747i_disco_board>` |                 | STM32H747XI | STM32H7       | STMicro STM32 | Arm Cortex-M7  | Arm          |
```

Comment:
```suggestion
   +--------------------------------------------------+-----------------+-------------+---------------+---------------+----------------+--------------+
   | :ref:`stm32h747i_disco <stm32h747i_disco_board>` | /stm32h747xx/m7 | STM32H747XI | STM32H7       | STMicro STM32 | Arm Cortex-M7  | Arm          |
  +--------------------------------------------------+-----------------+-------------+---------------+---------------+----------------+--------------+
   | |  /stm32h747xx/m4 | STM32H747XI | STM32H7       | STMicro STM32 | Arm Cortex-M4  | Arm          |
```

Reply:
stm32 has been removed as we have:
- `nrf5340dk` which shows the use of two cpuclusters where the cores in both clusters is of identical archtectures (cortex-m33 (arm))
- `mimx8mp_evk` which shows the use of two cpuclusters where the cores in the clusters are of different architectures (cortex-m7 (arm) / cortex-a53 (arm64))